src/meta/src/stream/stream_manager.rs

```diff
@@ -705,6 +712,151 @@ impl GlobalStreamManager {
 
         Ok(())
     }
+
+    async fn trigger_scale_out(&self, workers: Vec<WorkerId>) -> MetaResult<()> {
+        let _reschedule_job_lock = self.reschedule_lock.write().await;
+
+        let fragment_worker_changes = {
+            let guard = self.fragment_manager.get_fragment_read_guard().await;
+            let mut fragment_worker_changes = HashMap::new();
+            for table_fragments in guard.table_fragments().values() {
+                for fragment_id in table_fragments.fragment_ids() {
+                    fragment_worker_changes.insert(
+                        fragment_id,
+                        PbWorkerChanges {
+                            include_worker_ids: workers.clone(),
+                            ..Default::default()
+                        },
+                    );
+                }
+            }
+            fragment_worker_changes
+        };
+
+        let reschedules = self
+            .scale_controller
+            .generate_stable_resize_plan(
+                StableResizePolicy {
```

Comment:
Non-related to this PR, we may need to change the parallelism base on each job's parallelism setting: i.e. "AUTO" or a fixed number.